tracker/tests.py

```diff
@@ -1,3 +1,65 @@
-from django.test import TestCase
-
+from django.test import TestCase, Client
+from django.urls import reverse, resolve
+import json
+from .models import *
+from .views import *
+import datetime
 # Create your tests here.
```

Comment:
Have a look at https://pycqa.github.io/isort/ . It is a tool you can used to automatically sort the import to make your code cleaner.